src/test/java/org/cactoos/collection/BehavesAsCollection.java

```diff
@@ -62,26 +62,52 @@ public BehavesAsCollection(final E item) {
     @SuppressWarnings({ "unchecked", "PMD.ClassCastExceptionWithToArray" })
     public boolean matchesSafely(final Collection<E> col) {
         MatcherAssert.assertThat(
-            col, new IsCollectionContaining<>(new IsEqual<>(this.sample))
+            "Must contain item",
```

Comment:
Even though `MatcherAssert.assertThat` is still used, we are in transition to OO/Junit5 `new Assertion` for all the new code added.
Please use `new Assertion<>().affirm()`